src/horus/engine/driver/camera_usb.py

```diff
@@ -340,8 +358,6 @@ def set_brightness(self, value):
                     #value = int(value) / self._max_brightness
                     value = value * self._max_brightness / 255.
                     ret = self._capture.set(self.CV_CAP_PROP_BRIGHTNESS, value)
-                    if system == 'Linux' and ret:
-                        raise InputOutputError()
```

Comment:
These `InputOutputError` raises were, as far as I can tell, just incorrect.  If `ret` is `True`, that's opencv letting you know that the property _could_ be set; it's not clear to me why this logic existed :man_shrugging: .